src/modal/modal.ts

```diff
@@ -82,11 +82,12 @@ export class CaptureEyeModal extends LitElement {
     if (options.copyrightZoneTitle)
       this._copyrightZoneTitle = options.copyrightZoneTitle;
     if (
+      options.engagementZones &&
       JSON.stringify(options.engagementZones) !==
-      JSON.stringify(this._engagementZones)
+        JSON.stringify(this._engagementZones)
     ) {
       this._imageLoaded = false;
```

Comment:
Is it better to reset imageLoaded even when options.engagementZones is an empty array? For example, if you open Eye A with customized engagement zones, then open Eye B with no customized engagement zones (it should load the default one)